corehq/apps/groups/templates/groups/group_members.html

```diff
@@ -76,52 +76,43 @@
                 return;
             });
 
-            function success(name, id, gaEventLabel) {
+            function outcome(success, name, id, gaEventLabel) {
                 return function() {
-                    unsavedChanges[name] = false;
-                    $('.alert').removeClass('alert-error alert-info').addClass('alert-success');
-                    $('.alert')[0].innerHTML = name + " was successfully saved";
-                    $('.alert').show();
-                    $(id).find(':button').enableButton();
+                    var alertClass, message;
+                    if (success) {
+                        alertClass = 'alert-success';
+                        message = 'Successfully saved ' + name.toLowerCase() + '.';
+                        unsavedChanges[name] = false;
+                    }
```

Comment:
Not that I know of - github applies a fairly lenient linter to python code, but currently we don't have one for JS